lib/toolshed/unix.ex

```diff
@@ -27,12 +27,25 @@ defmodule Toolshed.Unix do
   @doc """
   Run a regular expression on a file and print the matching lines.
 
-  iex> grep ~r/video/, "/etc/mime.types"
+      iex> grep ~r/video/, "/etc/mime.types"
+
+  If colored is enabled, the matching lines will be highlighted.
+  You can also customize the highlighting by passing options supported
+  by `IO.ANSI.format/2`:
```

Comment:
99% of the time I would never use the formatting and be fine with the default. There might be times though where I need to grep errors that are already red in which I would use a different color (or invert it), and it seemed like an easy add to put that in the function.

That said, we could pick a different color as the default that might not typically be in default Elixir logger output, like blue or green. I picked red because that seems to be the default from the grep in shells

I agree that if it does stay, a `color:` option key would be better

Reply:
nvm - This is greping files which typically dont have color formatting. So defaulting to red for now until color is more thought through